<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>671</width>
      <height>411</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tab1">
     <attribute name="title">
      <string>Insert</string>
     </attribute>
     <widget class="QLineEdit" name="txtVertice">
      <property name="geometry">
       <rect>
        <x>470</x>
        <y>10</y>
        <width>101</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="addVertice">
      <property name="geometry">
       <rect>
        <x>580</x>
        <y>10</y>
        <width>71</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>AddVertice</string>
      </property>
     </widget>
     <widget class="CustomView" name="graphicsView">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>50</y>
        <width>641</width>
        <height>311</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnaddArista">
      <property name="geometry">
       <rect>
        <x>290</x>
        <y>10</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>AddArista</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtOrigen">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="txtDestino">
      <property name="geometry">
       <rect>
        <x>120</x>
        <y>10</y>
        <width>91</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLineEdit" name="peso">
      <property name="geometry">
       <rect>
        <x>230</x>
        <y>10</y>
        <width>51</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="label1">
      <property name="geometry">
       <rect>
        <x>110</x>
        <y>10</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
     <widget class="QLabel" name="label2">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>10</y>
        <width>16</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabDikjstra">
     <attribute name="title">
      <string>Dikjstra</string>
     </attribute>
     <widget class="QLineEdit" name="mensaje">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>40</y>
        <width>113</width>
        <height>23</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="btnEliminar">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>40</y>
        <width>80</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>eliminar</string>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>685</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>CustomView</class>
   <extends>QGraphicsView</extends>
   <header>customview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
